homeassistant/components/braviatv/__init__.py

```diff
@@ -64,31 +69,38 @@ class BraviaTVCoordinator(DataUpdateCoordinator[None]):
     several platforms.
     """
 
-    def __init__(self, hass, host, mac, pin, ignored_sources):
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        host: str,
+        mac: str,
+        pin: str,
+        ignored_sources: list[str],
+    ) -> None:
         """Initialize Bravia TV Client."""
 
         self.braviarc = BraviaRC(host, mac)
         self.pin = pin
         self.ignored_sources = ignored_sources
-        self.muted = False
-        self.channel_name = None
+        self.muted: bool | None = False
+        self.channel_name: str | None = None
         self.channel_number = None
```

Comment:
This one is missing type.

Reply:
This varable is not used so I'll remove it.